build.gradle

```diff
@@ -119,27 +119,37 @@ tasks.named('sonarqube').configure {
 }
 
 dockerCompose {
-    useComposeFiles = ["docker/docker-compose.yaml"]
-    startedServices = ['sge-container']
+    String targetEngine = System.getenv("GRID_ENGINE_API_TARGET_ENGINE") ?: "SGE"
```

Comment:
lets make things simpler. how about rename `GRID_ENGINE_API_TARGET_ENGINE` to `TARGET_GRID_ENGINE`